strongbox_test.go

```diff
@@ -41,7 +41,7 @@ func TestMultipleClean(t *testing.T) {
 	var doubleCleaned bytes.Buffer
 	clean(bytes.NewReader(cleaned.Bytes()), &doubleCleaned, "")
 
-	assert.Equal(cleaned.Bytes(), doubleCleaned.Bytes())
+	assert.Equal(string(cleaned.Bytes()), string(doubleCleaned.Bytes()))
```

Comment:
why the string conversions?

Reply:
https://github.com/uw-labs/strongbox/pull/14/commits/9e891842cffd713a4a7a1ff9ee1b16e844803e63